### PR TITLE
v2.0.0

### PR DESCRIPTION
**Pull request by douglasduteil**
---

- [ ] should allow to run on node@12

### CODE REVIEW

File: src/components/app/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import env from "@kosko/env";
+
+const { deployment, ingress, service } = create(env.component("app"))
+export default [deployment, ingress, service];
+```
+
+You can customize it
+
+```ts
+// in .k8s/components/_app.ts
```
Comment (revolunet) [inline src/components/app/README.md:19]: why `_app.ts` and not `app.ts` as above ?

File: src/components/app/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import { app } from "kubernetes-models/v1/app";
+import { create } from "@socialgouv/kosko-charts/components/app";
+import env from "@kosko/env";
+
+const params = env.component("app");
+const { deployment, ingress, service } = create(params)
+deployment.spec.template.spec.containers[0].name = "foo"
```
Comment (revolunet) [inline src/components/app/README.md:26]: one real life example on probes maybe ?

File: templates/simple/environments/_base/www.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      ? process.env.CI_COMMIT_TAG.slice(1)
+      : process.env.CI_COMMIT_SHA,
+  },
+
+  ingress: {
+    secretName: process.env.PRODUCTION ? "www-crt" : "wildcard-crt",
+  },
+
+  labels: {
+    component: "next",
```
Comment (revolunet) [inline templates/simple/environments/_base/www.ts:22]: `CI_PROJECT_NAME` ?

File: src/components/app/README.md (hunk 1 above)
Comment (douglasduteil) [inline src/components/app/README.md:19]:
```suggestion
// in .k8s/components/app.ts
```

Comment (douglasduteil) [inline src/components/app/README.md:19]:
```suggestion
// in .k8s/components/app.ts
```

File: templates/simple/environments/_base/www.ts (hunk 3 above)
Comment (douglasduteil) [reply templates/simple/environments/_base/www.ts:22]: Here `component` refers to [`app.kubernetes.io/component`](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels) which is a concrete description of what is running in the Pod.

**Comment by revolunet** [comment]
rédi

**Pull request merged**